capstone_project/src/main/java/com/google/sps/servlets/PostServlet.java

```diff
@@ -47,7 +49,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-    List<Entity> posts = datastore.prepare(query).asList(FetchOptions.Builder.withDefaults());
+    List<Entity> postEntities =
+        datastore.prepare(query).asList(FetchOptions.Builder.withDefaults());
+    List<Post> posts =
+        postEntities.stream().map(PostService::convertEntityToPost).collect(Collectors.toList());
+
```

Comment:
Shouldn't the response type be set here too?

Reply:
It's set on line 47. Do you mean moving down lower closer to where the the JSON is written to the response?

Reply:
oh nvm then.